test/tfile.c

```diff
@@ -8100,6 +8100,49 @@ test_min_dset_ohdr(void)
     CHECK(ret, FAIL, "H5Fclose");
 } /* end test_min_dset_ohdr() */
 
+/****************************************************************
+**
+**  test_unseekable_file():
+**    Test that attempting to open an unseekable file fails gracefully
+**    without a segfault (see hdf5#1498)
+****************************************************************/
+static void
+test_unseekable_file(void)
+{
+    hid_t file_id = H5I_INVALID_HID; /* File ID */
+
+    /* Output message about test being performed */
+    MESSAGE(5, ("Testing creating/opening an unseekable file\n"));
```

Comment:
Maybe fflush(stdout) after this, to know which test that causes a Segmentation Fault.